test/unit/Cardano/Wallet/DB/MVarSpec.hs

```diff
@@ -353,6 +386,22 @@ prop_createWalletTwice (key@(PrimaryKey wid), cp, meta) =
         runExceptT (createWallet db key cp meta) `shouldReturn` Right ()
         runExceptT (createWallet db key cp meta) `shouldReturn` Left err
 
+-- | Trying to create a same wallet twice should yield an error
```

Comment:
I suppose should be _Trying to remove the same wallet twice..._

Reply:
Indeed :+1: , wrong copy-pasting :grimacing:

Reply:
BTW. Perhaps the case _create wallet twice_ could be added here also? Or maybe it is not handled on the DB layer (I mean the fact _the same_ wallet cannot be created multiple times)?

Reply:
oh wait, it's already there :)